crates/bevy_ui/src/ui_transform.rs

```diff
@@ -170,6 +171,43 @@ impl UiGlobalTransform {
     pub fn try_inverse(&self) -> Option<Affine2> {
         (self.matrix2.determinant() != 0.).then_some(self.inverse())
     }
+
+    /// Creates a `UiGlobalTransform` from the given 2D translation.
+    #[inline]
+    pub fn from_translation(translation: Vec2) -> Self {
+        Self(Affine2::from_translation(translation))
+    }
+
+    /// Creates a `UiGlobalTransform` from the given 2D translation.
+    #[inline]
+    pub fn from_xy(x: f32, y: f32) -> Self {
+        Self::from_translation(Vec2::new(x, y))
+    }
+
+    /// Creates a `UiGlobalTransform` from the given rotation.
+    #[inline]
+    pub fn from_rotation(rotation: Rot2) -> Self {
+        Self(Affine2::from_angle(rotation.as_radians()))
```

Comment:
Doing
```suggestion
        Self(Affine2::from_mat2(rotation.into()))
```
Skips all floating point operations (notably trig and inverse trig).
That said we probably should offer a `From<Rot2> for Affine2` like we do with Isometry2d and Mat2